docs/zkapps/snarkyjs/snarkyjs-overview.mdx

```diff
@@ -0,0 +1,416 @@
+---
```

Comment:
I learned how to retain short titles in the sidebar and have useful titles on the page
https://github.com/o1-labs/docs2/wiki/Markdown-Syntax#sidebar